index.md

```diff
@@ -3,18 +3,18 @@ layout: default
 title: SapMachine
 ---
 
-**SapMachine** is a free, open-source, non-commercial, cross-platform, and fully-functional production-grade version of the Open Java Development Kit (OpenJDK).
-Built and supported by SAP, it provides a robust and reliable Java environment.
+**SapMachine** is the free, multiplatform, production-ready, and [Java SE certified](https://github.com/SAP/SapMachine/wiki/Certification-and-Java-Compatibility) [OpenJDK](https://openjdk.org/) (Open Java Development Kit) distribution by [SAP](https://sapmachine.io). It serves as the default JDK for SAP's countless applications and services.
 
```

Comment:
Erstmal vorab: vielen vielen Dank für den Fame.
Die Struktur ist gleich geblieben, die Sätze sind an sich gleich und an ähnlichen Stellen geblieben, ... das bedeutet, dass das ja gut so gewesen sein muss.
Was ich nicht verstehe und man möge es mir bitte mit einfachen Worten erklären: 
a) Wo ist der Bedarf jetzt wieder was zu ändern?
b) Wir haben 1000e Dinge die wichtiger wären als die Wortwahl der Webseite umzubauen. Warum tun wir das?
Die jetzige Seite ist bei weitem nicht so perfekt wie ich sie gerne hätte.
Wir bieten JMC zum download an und KEIN einziges Wort im Text was das ist. Ich hatte es damals schlicht vergessen. Mit Dir Johannes habe ich darüber gesprochen. => c) warum steht auch hier wieder nicht auch nur ein einziger Satz von JMC?
d) nachdem der wiederbelebte SapMachine Newsletter raus kam wolltest Du einen Supportability Tools Newsletter haben. Weil es ja so viel zu Supportability zu berichten gibt. Du hast Dir selbst eine Kadenz von alle 2 oder alle 3 Monate auferlegt. Wir hatten jetzt einen einzigen dieser Newsletter. Weil "es gibt aktuell nichts zu supportability!". 
Du wolltest vor dem CPU in Deiner "guten Idee" einen ausführlichen Text über async-profiler schreiben und den dann über diesen Newsletter verschicken. VOR dem CPU. Den recyclen wir dann im SapMachine Newsletter. Das war der Plan. Bis heute kam kein Text in besagtem Newsletter. d) Warum gibt es diesen Newsletter wenn selbst wenn es berichtenswerte Sachen nicht über diesen Supportability Tools Newsletter verteilt werden?
...
m) warum geht der Link auf diesem PullRequest mit dem Text "SAP" nicht zu SAP?
n) der besagte Link geht auf sich selbst. Was war die Intention diesen Link so zu setzen?

Es ist vor dem ersten Kaffee, deswegen bitte mit wenigen einfachen Worten beantworten.
Ich versteh' es nicht

Reply:
> Erstmal vorab: vielen vielen Dank für den Fame.
> Die Struktur ist gleich geblieben, die Sätze sind an sich gleich und an ähnlichen Stellen geblieben, ... das bedeutet, dass das ja gut so gewesen sein muss.

Dein ursprünglicher Text war gut. Ich habe ihn mit ein paar Links angereichert, mit dem Text von Coretto abgeglichen und etwas konkretiesiert.

> a) Wo ist der Bedarf jetzt wieder was zu ändern?

Wir hatten gestern über unsere SEO Probleme geredet und ich hatte gestern ein Gespräch mit einem potentiellen New-Hire, der nach dem Lesen der Website nicht wirklich wusste, was die SapMachine ist. Deswegen habe ich Christoph gefragt, ob ich am Nachmittag an der Website arbeiten darf und er hat sein Ok gegeben.

> Wir bieten JMC zum download an und KEIN einziges Wort im Text was das ist. Ich hatte es damals schlicht vergessen. Mit Dir Johannes habe ich darüber gesprochen. => c) warum steht auch hier wieder nicht auch nur ein einziger Satz von JMC?

Weil wir keine signifikanten contributions zum JMC gemacht haben, deswegen erwähne ich ihn nicht im Satz mit unseren Contributions. Ich habe ihn jetzt weiter unter erwähnt. Danke für deinen Hinweis.

> d)
Was hat das mit diesem PR zu tun?

m) warum geht der Link auf diesem PullRequest mit dem Text "SAP" nicht zu SAP?

Danke für den Hinweis, ist behoben.